pkg/relayer/miner/gen/gen_fixtures.go

```diff
@@ -84,7 +94,8 @@ func main() {
 	randRelaysObs, errCh := genRandomizedMinedRelayFixtures(
 		ctx,
 		defaultRandLength,
-		miner.DefaultRelayHasher,
+		// TODO_TECHDEBT: need to find out how the removal of miner.DefaultRelayerHasher in commit 608edc31b39c6ff9a285af46d1c1037cd36a7252 did not
```

Comment:
Thanks, deleted the TODO_TECHDEBT comment and updated the Makefile to generate the relay fixtures under `go develop` target. This will ignore the file in builds, but help catch potential errors when making changes.